restaurants/src/app/components/restaurantedit/restaurant-edit/restaurant-edit.component.html

```diff
@@ -0,0 +1,14 @@
+<div *ngIf="restaurant">
+    <h2>{{restaurant.name | uppercase}} EDIT Details</h2>
+    <div>
+        <label for="name">Restaurant name:</label>
+        <input id="name" [(ngModel)]="restaurant.name" placeholder="name"><br>
+    
+        <label for="rank">Rank:</label>
+        <input id="rank" [(ngModel)]="restaurant.rank" placeholder="rank"><br>
+    
+        <button type="button" (click)="save(restaurant.name, restaurant.rank)">Save</button>
```

Comment:
Is it necessary to pass `restaurante.name` and `restaurant.rank` as parameters to the `save()` method? You are already using two-way binding on the inputs to those variables using `[(ngModel)]` - so `save()` should already have access to those vars